packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.types.ts

```diff
@@ -177,6 +177,12 @@ export interface ICoachmarkProps extends React.ClassAttributes<CoachmarkBase> {
    */
   preventDismissOnLostFocus?: boolean;
 
+  /**
+   * If true then the Coachmark beak will always be visible as long as Coachmark is visible
+   * @defaultvalue false
+   */
+  persistentBeak?: boolean;
```

Comment:
Nit: I realize this is prior art, but it might be nice to clarify what we mean by `beak`, as it's not clearly defined in the existing docs for coachmark.

Reply:
updated